src/database/book_copy.sql

```diff
@@ -0,0 +1,10 @@
+
+-- ---------------------------------------------------------------------------
+--      BOOK COPY TABLE. STORES THE DETAILS OF A BOOK_COPY
+-- ---------------------------------------------------------------------------
+
+CREATE TABLE BOOK_COPY (
+ID BIGINT NOT NULL  AUTO_INCREMENT, -- BOOK COPY ID
+BOOK_ID BIGINT NOT NULL, -- the book id. Foreign key to book table(ben)
```

Comment:
You need to add foreign key
ALTER TABLE BOOK_COPY ADD FOREIGN KEY (BOOK_ID) REFERENCES BOOK(ID) ;
OR
at the time of table creation
CREATE TABLE BOOK_COPY (
 ID BIGINT NOT NULL  AUTO_INCREMENT, -- BOOK COPY ID
 BOOK_ID BIGINT NOT NULL, -- the book id. Foreign key to book table(ben)
 PRIMARY KEY (ID),
  FOREIGN KEY (BOOK_ID)
        REFERENCES BOOK(ID)
 );

remember, once you have data, you have no longer the option of dropping and creating the table and hence always it would be alter table.

Reply:
Chose second option. Done